sound/soc/msm/msm8x16.c

```diff
@@ -51,13 +51,12 @@
 #define DEFAULT_MCLK_RATE 9600000
 
 #define EXT_CLASS_D_EN_DELAY 13000
-#define EXT_CLASS_D_DIS_DELAY 3000
```

Comment:
Disabling GPIO takes less time than enabling it.
That's why we use different delay times.